backend/de.metas.banking/de.metas.banking.swingui/src/main/java/de/metas/banking/payment/paymentallocation/model/PaymentRow.java

```diff
@@ -242,14 +244,15 @@ public IPaymentDocument copyAsPaymentDocument()
 		final CurrencyId currencyId = currenciesRepo.getByCurrencyCode(paymentRow.getCurrencyISOCode()).getId();
 
 		return PaymentDocument.builder()
-				.orgId(paymentRow.getOrgId())
+				.clientAndOrgId(ClientAndOrgId.ofClientAndOrg(ClientId.METASFRESH, paymentRow.getOrgId()))
```

Comment:
avoid avoid hardcoding ClientId.
ClientId.METASFRESH is manly for testing!

Reply:
Problem: i don't have a client id here and i believe org is null as well as it is never set to the builder. 
I guess this never failed because it was never used.
Is there a point in investing any time in fixing this, since this belongs to swing?

Can i use `Env.getClientId()` and `Env.getOrgId()` to get some generic values?

Reply:
agree. don't invest time in swing.

Reply:
fixed